debian/rules

```diff
@@ -39,6 +39,16 @@ else
     TESTSUITE_FAIL_CMD:=exit 1
 endif
 
+# RocksDB cannot build on 32-bit platforms
+ifeq (32,$(DEB_HOST_ARCH_BITS))
+    CMAKEFLAGS += -DWITHOUT_ROCKSDB=true
+endif
+
+# ColumnStore only attempts to build on a few platforms as dictated by CMake checks
```

Comment:
I might have missed that but with this transition we allow MCS to be build even with Travis CI pipelines that Sergei wanta to avoid.

Reply:
We cannot build MCS on Travis as it will be too slow. Related https://jira.mariadb.org/browse/MCOL-4117

Reply:
We had and still have in autobake code that prevents Travis-CI from building MCS.